cranelift/codegen/src/machinst/lower.rs

```diff
@@ -377,12 +377,33 @@ impl<'func, I: VCodeInst> Lower<'func, I> {
             }
         }
 
-        // Make a sret register, if one is needed.
+        // Find the sret register, if it's used.
         let mut sret_reg = None;
         for ret in &vcode.abi().signature().returns.clone() {
```

Comment:
I don't think the `returns` array needs to be cloned now. I think after the changes in this PR, the loop body only reads from `vcode`.